dev/breeze/src/airflow_breeze/utils/run_utils.py

```diff
@@ -40,12 +41,22 @@ def run_command(
     verbose: bool = False,
     suppress_raise_exception: bool = False,
     suppress_console_print: bool = False,
+    env: Optional[Mapping[str, str]] = None,
     **kwargs,
 ):
     if verbose:
-        console.print(f"[blue]$ {' '.join(shlex.quote(c) for c in cmd)}")
+        command_to_print = ' '.join(shlex.quote(c) for c in cmd)
+        # if we pass environment variables to execute, then
+        env_to_print = ' '.join(f'{key}="{val}"' for (key, val) in env.items()) if env else ''
+        # Soft wrap allows to copy&paste and run resulting output as it has no hard EOL
+        console.print(f"[blue]{env_to_print} {command_to_print}[/]", soft_wrap=True)
```

Comment:
This one is interesting. Originally rich "hard" wraps the lines in console - so when you copy & paste such output, you get line breaks and if you have long line the copy&paste will fail. 

The "soft_wrap" parameter of print, removes the hard line breaks, and now you can simply select the command printed in blue, copy &paste it and it should nicely work :).